src/java.base/aix/native/libnio/ch/Pollset.c

```diff
@@ -1,6 +1,6 @@
 /*
- * Copyright (c) 2008, 2013, Oracle and/or its affiliates. All rights reserved.
- * Copyright (c) 2012 SAP SE. All rights reserved.
+ * Copyright (c) 2008, 2024, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2012, 2024 SAP SE. All rights reserved.
```

Comment:
It seems you only need to update the copyright of the company you work for.

Reply:
Oracle requests/requires that the Oracle copyright always be updated when a file is modified.

Reply:
> Oracle requests/requires that the Oracle copyright always be updated when a file is modified.

Got it. Thanks for your explanation.